vaadin-time-picker-flow-parent/vaadin-time-picker-flow/src/main/java/com/vaadin/flow/component/timepicker/TimePicker.java

```diff
@@ -59,11 +66,18 @@
  *
  * @author Vaadin Ltd
  */
+@Tag("vaadin-time-picker")
+@NpmPackage(value = "@vaadin/polymer-legacy-adapter", version = "24.0.0-alpha6")
+@JsModule("@vaadin/polymer-legacy-adapter/style-modules.js")
+@NpmPackage(value = "@vaadin/time-picker", version = "24.0.0-alpha6")
+@JsModule("@vaadin/time-picker/src/vaadin-time-picker.js")
 @JsModule("./vaadin-time-picker/timepickerConnector.js")
-public class TimePicker extends GeneratedVaadinTimePicker<TimePicker, LocalTime>
-        implements HasSize, HasValidation, HasEnabled, HasHelper, HasLabel,
-        HasTheme, HasClearButton, HasAllowedCharPattern,
-        HasValidator<LocalTime>, HasClientValidation, HasTooltip {
+public class TimePicker
+        extends AbstractSinglePropertyField<TimePicker, LocalTime>
+        implements Focusable<TimePicker>, HasAllowedCharPattern, HasClearButton,
+        HasClientValidation, HasEnabled, HasHelper, HasLabel, HasSize, HasStyle,
```

Comment:
`HasEnabled` is now redundant:

<img width="919" alt="Screenshot 2022-12-13 at 13 27 11" src="https://user-images.githubusercontent.com/1222264/207306128-5a6cf5cf-aa63-4f49-a979-ab5a40f97a95.png">

Reply:
Removed.